tests/test_select.py

```diff
@@ -16,11 +16,13 @@ def test_select_with_schema_and_database():
 
 
 def test_select_with_table_name_in_backtick():
-    assert_table_lineage_equal("SELECT * FROM `tab1`", {"tab1"})
+    assert_table_lineage_equal("SELECT * FROM `tab1`", {"tab1"}, dialect="bigquery")
 
 
 def test_select_with_schema_in_backtick():
-    assert_table_lineage_equal("SELECT col1 FROM `schema1`.`tab1`", {"schema1.tab1"})
+    assert_table_lineage_equal(
+        "SELECT col1 FROM `schema1`.`tab1`", {"schema1.tab1"}, dialect="bigquery"
+    )
```

Comment:
We can use pytest parametrize functionality to test these two cases. backtick for quoting identifier is also a routine from MySQL as well as BigQuery.
```python
@pytest.mark.parametrize("dialect", ["mysql", "bigquery"])
def test_select_with_schema_in_backtick(dialect):
```